.github/template/.gitignore

```diff
@@ -0,0 +1 @@
+vercel.json
```

Comment:
This is needed to ignore the copy of `vercel.json` file when using the template.